library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -344,6 +347,12 @@ public WeekView(Context context, AttributeSet attrs, int defStyleAttr) {
             mShowNowLine = a.getBoolean(R.styleable.WeekView_showNowLine, mShowNowLine);
             mHorizontalFlingEnabled = a.getBoolean(R.styleable.WeekView_horizontalFlingEnabled, mHorizontalFlingEnabled);
             mVerticalFlingEnabled = a.getBoolean(R.styleable.WeekView_verticalFlingEnabled, mVerticalFlingEnabled);
+            mCustomTypefacePath = a.getString(R.styleable.WeekView_customTypefacePath);
+
+            // create typeface from assets folder e.g "fonts/..."
+            if(!TextUtils.isEmpty(mCustomTypefacePath)){
+                mCustomTypeface = Typeface.createFromAsset(getContext().getAssets(),mCustomTypefacePath);
```

Comment:
This can cause an exception if user provides an invalid path in `mCustomTypefacePath`